runtime/visualize.cc

```diff
@@ -475,6 +473,18 @@ function draw(mem, t) {
   window.requestAnimationFrame(function(t) { draw(mem, __current_t); });
 }
 function check(condition) {}
+function or_else(...cond) {
```

Comment:
This evaluates every argument before calling the function, so it doesn't short circuit. Just add a TODO for now, I don't think it's worth the complexity to fix this, it's just the visualizer and it's only used to avoid integer overflow at the moment.

Reply:
I forgot this was generated code! But that's fine still.